### PR TITLE
Fixed supply temperature not set correctly.

### PR DESCRIPTION
**Pull request by samvanderzwan**

### CODE REVIEW

File: unit_test/entities/test_esdl_object.py [hunk 1]
```diff
@@ -173,6 +173,50 @@ def test_get_return_temperature_out(self):
         # Assert
         self.assertEqual(return_temperature, 313.15)
 
+    def test_get_supply_temperature_consumer(self):
```
Comment (vanmeerkerk) [inline unit_test/entities/test_esdl_object.py:176]: I don't know whether this test will cover what is required. You changed the input/output defintion of the Consumer/Producer asset but the tests did not fail; does this test now ensure that when you change that defintion this test will fail?

**Review by vanmeerkerk — CHANGES_REQUESTED** [review]
See single comment on testing. I doubt whether the test is now complete enough to cover the problem of not setting the correct definitions.

File: unit_test/entities/test_esdl_object.py (hunk 1 above)
Comment (samvanderzwan) [reply unit_test/entities/test_esdl_object.py:176]: Basically as I checked we do not have proper test for the controller to entity method. These should be added I created new issue for that. #191

**Review by vanmeerkerk — APPROVED** [review]
Discussed.

**Pull request merged**